sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -75,11 +75,13 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]]
 
   override val origin: Origin = CurrentOrigin.get
 
-  /**
-   * A mutable map for holding auxiliary information of this tree node. It will be carried over
-   * when this node is copied via `makeCopy`, or transformed via `transformUp`/`transformDown`.
-   */
```

Comment:
We probably want to preserve some version of this comment so that we continue to provide context to future readers?